test/api_test.js

```diff
@@ -295,6 +296,29 @@ vows
             assert.typeOf(header, 'string');
           });
         }
+      },
+
+      "removeAllCookiesSync": {
+        topic: function () {
+          var jar = new CookieJar();
+          var cookie1 = Cookie.parse("a=b; Domain=example.com; Path=/");
+          var cookie2 = Cookie.parse("a=b; Domain=foo.com; Path=/");
+          var cookie3 = Cookie.parse("foo=bar; Domain=foo.com; Path=/");
+          jar.setCookieSync(cookie1, 'http://example.com/index.html');
+          jar.setCookieSync(cookie2, 'http://foo.com/index.html');
+          jar.setCookieSync(cookie3, 'http://foo.com/index.html');
+
+          jar.removeAllCookiesSync();
+          
+          return jar;
+        },
+        "no cookies in the jar": function (jar) {
+          jar.store.getAllCookies(function(err, cookies) {
+            assert(err == null);
+            assert(cookies != null);
+            assert(cookies.length === 0, 'cookies were not removed')
```

Comment:
Typo: should be "cookies were removed"

Reply:
this message should specify the reason why cookies array is not empty. Isn't it?